perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -846,6 +846,20 @@ public RichMember getRichMemberWithAttributes(PerunSession sess, Member member)
 		return getPerunBl().getMembersManagerBl().filterOnlyAllowedAttributes(sess, getMembersManagerBl().getRichMembersWithAttributes(sess, vo, status), null, true);
 	}
 
+	@Override
+	public List<RichMember> getRichMembersNoUserAttributes(PerunSession sess, Vo vo) throws PrivilegeException, VoNotExistsException {
+		Utils.checkPerunSession(sess);
+
+		getPerunBl().getVosManagerBl().checkVoExists(sess, vo);
+
+		// Authorization
+		if (!AuthzResolver.authorizedInternal(sess, "getRichMembers_Vo_policy", vo)) {
+			throw new PrivilegeException(sess, "getRichMembersNoUserAttributes");
+		}
+
+		return getPerunBl().getMembersManagerBl().filterOnlyAllowedAttributes(sess, getMembersManagerBl().getRichMembers(sess, vo), null, true);
```

Comment:
```suggestion
		return getPerunBl().getMembersManagerBl().filterOnlyAllowedAttributes(sess, getMembersManagerBl().getRichMembersNoUserAttributes(sess, vo), null, true);
```
we need to call the right method